src/bidiMapper/domains/script/channelProxy.ts

```diff
@@ -16,13 +16,13 @@
  *
  */
 
-import {CommonDataTypes, Script} from '../../../protocol/protocol.js';
+import {type CommonDataTypes, Script} from '../../../protocol/protocol.js';
 import type {IEventManager} from '../events/EventManager.js';
 import {uuidv4} from '../../../utils/uuid';
 
 import type {Realm} from './realm.js';
 
-import Handle = CommonDataTypes.Handle;
+type Handle = CommonDataTypes.Handle;
```

Comment:
What change does this yield? Is it a no-op?

Reply:
It's a no-op, brakes some building (esbuild) and does nothing but add alias. To be honest don't know how it works at all. But it's just a short name, should be in the import statement, will update.